sdk/eventhub/eventhubs-checkpointstore-tables/src/log.ts

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { createClientLogger } from "@azure/logger";
+
+/**
+ * The `@azure/logger` configuration for this package.
+ * This will output logs using the `azure:eventhubs-checkpointstore-blob` namespace prefix.
```

Comment:
Make sure comments are updated to refer to tables, not blob :smile:
```suggestion
 * This will output logs using the `azure:eventhubs-checkpointstore-tables` namespace prefix.
```